Project/Versie_3/style.css

```diff
@@ -36,11 +36,11 @@ body {
   display: -ms-flexbox;
   display: flex;
   -webkit-box-pack: justify;
-      -ms-flex-pack: justify;
-          justify-content: space-between;
+  -ms-flex-pack: justify;
```

Comment:
just adding a comment
```suggestion
  -ms-flex-pack: justify;
```

Reply:
resolved as this is just a comment. No spamming please..